components/dropbox/sources/new-file/new-file.mjs

```diff
@@ -1,46 +1,47 @@
-const dropbox = require("../../dropbox.app.js");
+import common from "../common.mjs";
 
-module.exports = {
+export default {
+  ...common,
   key: "dropbox-new-file",
   name: "New File",
-  version: "0.0.4",
-  description:
-    "Emits an event when a new file is added to your account or a specific folder. Make sure the number of files/folders in the watched folder does not exceed 4000.",
+  version: "0.0.5",
+  description: "Emits an event when a new file is added to your account or a specific folder. Make sure the number of files/folders in the watched folder does not exceed 4000.",
```

Comment:
The eslint rules like it if source descriptions start with "Emit new".
`description: "Emit new event when a new file is added to your account or a specific folder. Make sure the number of files/folders in the watched folder does not exceed 4000.",`

Also, there should be a type here:
`type: "source",`